src/VuetifyResource.vue

```diff
@@ -738,6 +740,33 @@
                     return this.texts[t];
                 }
             },
+
+            canDeleteResources(selected) {
+                let canDelete = this.canDelete;
+                if (this.canDeleteResourceKey !== '') {
+                    selected.forEach((resource) => {
+                        if (resource[this.canDeleteResourceKey] === false) {
+                            canDelete = false;
+                            return;
```

Comment:
Not sure, but isn't it possible to just `return false;` ? or does forEach have its own scope?

Reply:
Correct, forEach has it's own scope